client/packages/host/src/Admin/ElectronSettings.tsx

```diff
@@ -93,47 +98,73 @@ export const ElectronSettings = () => {
       <Typography variant="h5" color="primary" style={{ paddingBottom: 25 }}>
         {t('heading.barcode-scanners')}
       </Typography>
+
       <Setting
-        title={t('label.current')}
-        component={
-          !scanner ? (
-            <Box display="flex">
-              <AlertIcon color="error" />
-              <Typography
-                component="div"
-                sx={{
-                  alignItems: 'center',
-                  display: 'inline-flex',
-                  fontSize: '14px',
-                  paddingLeft: 1,
-                }}
-              >
-                {t('messages.no-scanners-found')}
-              </Typography>
-            </Box>
-          ) : (
-            <Scanner scanner={scanner} />
-          )
-        }
-      />
-      <Setting
-        title=""
+        title={'Scanner Type'}
         component={
-          <Box>
-            {isScanning && (
-              <Box style={{ textAlign: 'center' }} padding={2}>
-                <img src={omsBarcode} alt="omSupply Barcode" />
-                <Typography>{t('messages.detect-scanner')}</Typography>
-              </Box>
-            )}
-            <Box display="flex" justifyContent="flex-end">
-              <LoadingButton onClick={startDeviceScan} isLoading={isScanning}>
-                {t('label.detect-scanners')}
-              </LoadingButton>
+          <Box display="flex" justifyContent="flex-end" alignItems="center">
+            <Switch
+              label={'USB Serial'}
+              checked={scannerType === 'usb_keyboard'}
```

Comment:
This is weird (to have opposite in checked and label), but I think this is how the Switch between two values works in other places

Reply:
I made an issue: https://github.com/openmsupply/open-msupply/issues/1813